tools/cli/admin.go

```diff
@@ -1002,6 +1003,11 @@ func getDBFlags() []cli.Flag {
 			Name:  FlagTLSEnableHostVerification,
 			Usage: "cassandra tls verify hostname and server cert (tls must be enabled)",
 		},
+		cli.GenericFlag{
+			Name:  FlagConnectionAttributes,
+			Usage: "a key-value set of sql database connection attributes (must be in key=value format)",
+			Value: &flag.StringMap{},
```

Comment:
Nit: what do you think about keeping consistent with the sql-tool?
https://github.com/uber/cadence/blob/d5fbc5e3b9c978f7ba9b1c93cb162e261df6b650/tools/sql/handler.go#L171
The format is URL encoded: `k1=v1&k2=v2`

Reply:
I didn't realize the flag is already defined in `sql-tool`. I'll update this PR to keep them consistent with each other.

Reply:
Thanks! Can you also help update the usage in the sql-tool? that's also missing the details

Reply:
Sure, updated!